localstack/services/cloudformation/engine/template_deployer.py

```diff
@@ -182,7 +182,14 @@ def resolve_refs_recursively(
     value,
 ):
     result = _resolve_refs_recursively(
-        account_id, region_name, stack_name, resources, mappings, conditions, parameters, value
+        account_id,
+        region_name,
+        stack_name,
+        resources,
+        mappings,
+        conditions,
+        parameters,
+        value,
```

Comment:
nit: these formatting changes seem to dominate the PR diff. Is that because of our switching to `ruff`? If not can we remove these formatting changes to clean up the PR?